packages/editor/src/components/parts/cache/CacheLifetime.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from 'test-utils';
+import { CacheLifetime } from './CacheLifetime';
+
+describe('CacheLifetime', () => {

```

Comment:
add a test here, that the lifetime input does not exist when the radio "forever" is selected